opentracing-api/src/main/java/io/opentracing/ext/Constants.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.ext;
+
+public class Constants {
```

Comment:
What are these constants for? I can't seem to see any use or reason for them. Do they actually belong in this PR?

"Constants" classes are an anti-pattern. It's better to have them, if possible, in the class that actually owns and is responsible for exposing them (if it exists). The exception is if this is truly a package level constant, which is a rare thing.